frontend/src/components/problem/Selector.tsx

```diff
@@ -136,17 +153,17 @@ export function ProblemSelector(props: ProblemSelectorProps) {
       <TextSearch
         onClick={() => setShowProblems(!showProblems)}
         onChange={setSearchStatus}
-        placeholder={problems.length ? 'Select problems (optional)' : 'Loading...'}
+        placeholder={allProblems.length ? 'Search by name, difficulty, etc.' : 'Loading...'}
```

Comment:
This change turns out makes the ending portion of the placeholder not visible - could you extend the input so the placeholder is fully visible? Maybe make the `width` attribute `100%` or something similar?

**Filter Input Placeholder**
<img width="1278" alt="Filter Input Placeholder" src="https://user-images.githubusercontent.com/7987279/121825577-12dbec80-cc68-11eb-9dfb-8680dd1f28ad.png">

Reply:
My slight concern about this would be that this would still be slightly cut off on smaller screen sizes (anything below 1075px from quick test), and for large screen sizes, the dropdown might get a little out of proportion:

<img width="1047" alt="Problems are very wide" src="https://user-images.githubusercontent.com/24768574/121825768-0dcb6d00-cc69-11eb-93fb-8bc101622434.png">

Thoughts? It'd be nice to be able to scroll that text, although idk if that's possible for a placeholder text

Reply:
Any preference on this? I don't think it's too big of a deal, but among the options available I might lean towards leaving it as is (text cut off) or reverting the text to the original

Reply:
Ah sorry just saw now - yeah that's definitely a valid concern there, we should keep the input the same width. Maybe change to `Filter by name, tag, etc. (separate queries by comma)` - just removing the `difficulty` one which they can probably discover themselves so that it fits the whole phrase? Or any of the other options tbh, this isn't the biggest deal.

Reply:
Hm ok I might just leave it as is for now then (since it's a pretty small detail and so as not to make another commit), and maybe in a future PR if either of us remembers or wants to change the text we can always update it then